Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -352,6 +352,9 @@ def parse_args(args):
         optparse.make_option(
             "--use-gpu-process", action="store_true", default=False,
             help=("Enable all GPU process related features, also set additional expectations and the result report flavor.")),
+        optparse.make_option(
+            "--site-isolation", action="store_true", default=False,
+            help=("Run the test in a cross origin iframe with and without site isolation enabled and compare the results. Uses site-isolation test expectations")),
```

Comment:
nit: is this "Run **each** test in a cross origin iframe..."? How does this deal with reftests?

Reply:
Changing to each.  I'm not 100% sure how it handles reftests, but I think it does the same as all the other tests: run in an iframe with site isolation off, get output, run in an iframe with site isolation on, get output, compare.